packages/backend/src/ee/database/migrations/20251022131031_add_chart_fields_to_embeddings.ts

```diff
@@ -0,0 +1,47 @@
+import { Knex } from 'knex';
+
+const EMBEDDING_TABLE_NAME = 'embedding';
+const fields = {
+    chart_uuids: 'chart_uuids',
+    allow_all_charts: 'allow_all_charts',
+};
+export async function up(knex: Knex): Promise<void> {
+    if (
+        !(await knex.schema.hasColumn(EMBEDDING_TABLE_NAME, fields.chart_uuids))
+    ) {
+        await knex.schema.alterTable(EMBEDDING_TABLE_NAME, (table) => {
+            table
+                .specificType(fields.chart_uuids, 'text[]')
```

Comment:
do we need to add index for these new fields ? are we going to do any search on these ? 

This table should have minimal impact anyway, there is at most 1 entry per org per project